client/src/components/TreeDropdownField/TreeDropdownField.js

```diff
@@ -343,7 +246,7 @@ class TreeDropdownField extends Component {
     }
 
     // If ancestor node is already loaded (and non-empty) then don't re-trigger
-    const foundTree = this.findTreeByPath(this.props.tree, path);
+    const foundTree = findTreeByPath(this.props.tree, path);
```

Comment:
shouldn't this be `this.props.findTreeByPath()`? Seems weird to be from props in some parts and not in other parts